index.html

```diff
@@ -509,7 +509,10 @@ <h2>Credentials</h2>
 same <a>entity</a>. It may include an identifier as well as metadata that
 describes properties of the credential itself such as: the <a>issuer</a>,
 the expiry time, a representative image, a public key to use for the purposes
-of verification, the revocation mechanism to use etc.
+of verification, the revocation mechanism to use, etc.
+(Where images are used, it is recommended to consider accessibility
+issues as described in <a href="https://www.w3.org/WAI/tutorials/images/complex/">W3C
```

Comment:
This isn't what Janina and the Accessibility folks were requesting (they wanted descriptions of the images used in the W3C VC specification), so suggest that we strike it (but keep the rest of the PR).

Reply:
@dhh1128 is raising a good concern related to expressing VCs that contain images to people with Accessibility needs. I'm not certain we have enough experience to say much at this point other than something to this effect: "If you use images, you're making future Accessibility concerns more difficult... stick to just expressing pure data in VCs so that systems can choose how to render the VC appropriately (e.g. audio, braille, high contrast graphics, vibrations, etc.)."